content/applications/inventory_and_mrp/manufacturing/basic_setup/configure_manufacturing_product.rst

```diff
@@ -24,6 +24,35 @@ checkbox in the :guilabel:`Routes` section. This tells Odoo the product can be m
    :align: center
    :alt: The Manufacturing route on the Inventory tab of a product page.
 
+.. _manufacturing/basic_setup/lot-serial-tracking:
+
+Lot/serial number tracking
+--------------------------
+
```

Comment:
The assignment of lots or serial numbers to newly manufactured products is **optional**.

Reply:
That could be used as a lead-in sentence to this section to *really drive home* that point -- but that's completely up to you

Reply:
This is a great suggestion! Will go with this